src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const splittedString = sourceString.split(';');
+  const result = {};
```

Comment:
The variable name 'result' is too generic. It's better to use a more descriptive name like 'styleObject' to clearly indicate its purpose as the object that will hold the CSS properties and values.